### PR TITLE
Update release tools

### PR DESCRIPTION
**Pull request by saad-ali**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Update release-tools to head of https://github.com/kubernetes-csi/csi-release-tools/ master branch

Changes go build version from 1.12.4 to 1.13.3

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by msau42** [comment]
CI failure is due to two reasons:
1) resizer has [overridden](https://github.com/kubernetes-csi/external-resizer/blob/master/.prow.sh#L3) e2e versions to use k8s latest. I think we should remove it.
2) hostpath deployment version is fixed to 1.2.0 which doesn't support beta snapshots yet. We can cut an rc tag and update the [hostpath version](https://github.com/kubernetes-csi/external-resizer/pull/68/files#diff-d090135c7b07955c3c2303b588b5dfcfR190) that the test uses

File: release-tools/prow.sh [hunk 1]
```diff
@@ -183,7 +187,7 @@ configvar CSI_PROW_WORK "$(mkdir -p "$GOPATH/pkg" && mktemp -d "$GOPATH/pkg/csip
 #
 # When no deploy script is found (nothing in `deploy` directory,
 # CSI_PROW_HOSTPATH_REPO=none), nothing gets deployed.
-configvar CSI_PROW_HOSTPATH_VERSION "v1.2.0-rc8" "hostpath driver"
+configvar CSI_PROW_HOSTPATH_VERSION "v1.2.0" "hostpath driver"
```
Comment (msau42) [inline release-tools/prow.sh]: I just tagged an rc for hostpath repo which has the beta snapshots changes. Can you try setting v1.3.0-rc2 here?

Comment (saad-ali) [reply release-tools/prow.sh]: Will do!

**Comment by msau42** [comment]
The tests ran but results got clobbered.  Investigating...

**Comment by saad-ali** [comment]
Rebased with https://github.com/kubernetes-csi/csi-release-tools/pull/60 and https://github.com/kubernetes-csi/csi-release-tools/pull/61 merged.

All green now.

PTAL

**Pull request merged**